assets/js/atomic/blocks/product-elements/title/index.ts

```diff
@@ -45,7 +46,7 @@ const blockConfig: BlockConfiguration = {
 				gradients: true,
 				__experimentalSkipSerialization: true,
 			},
-			...( hasSpacingStyleSupport() && {
+			...( __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
The `tyepeof` keyword is missing:

```suggestion
			...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```